log.capnp

```diff
@@ -742,6 +743,19 @@ struct ControlsState @0x97ff69c53601abf1 {
     steeringAngleDesiredDeg @4 :Float32;
   }
 
+  struct LateralCurvatureState {
+    active @0 :Bool;
+    actualCurvature @1 :Float32;
+    desiredCurvature @2 :Float32;
+    error @3 :Float32;
+    p @4 :Float32;
+    i @5 :Float32;
+    d @6 :Float32;
```

Comment:
are you using any derivative?

Reply:
Not right now, and curvature controller as is doesn't calculate the rate so doesn't support it